src/sentry/utils/pytest/sentry.py

```diff
@@ -192,6 +192,10 @@ def pytest_configure(config):
     # this isn't the real secret
     settings.SENTRY_OPTIONS["github.integration-hook-secret"] = "b3002c3e321d4b7880360d397db2ccfd"
 
+    # Configure control backend settings for storage
+    settings.SENTRY_OPTIONS["filestore.control.backend"] = "filesystem"
+    settings.SENTRY_OPTIONS["filestore.control.options"] = {"location": "/tmp/sentry-files"}
```

Comment:
Not a fan of having this config duplicated from `defaults.py` so open to suggestions on better places to set this

Reply:
I think the duplication is fine. Test config often diverges from other config data.